src/Build.UnitTests/Evaluation/Expander_Tests.cs

```diff
@@ -3682,6 +3682,36 @@ public void PropertyFunctionWithNewLines()
             result.ShouldBe("6C854");
         }
 
+        [Fact]
+        public void PropertyFunctionStringIndexOfAny()
+        {
+            TestPropertyFunction("$(prop.IndexOfAny('y'))", "prop", "x-y-z", "2");
+        }
+
+        [Fact]
+        public void PropertyFunctionStringLastIndexOf()
+        {
+            TestPropertyFunction("$(prop.LastIndexOf('y'))", "prop", "x-x-y-y-y-z", "8");
+
+            TestPropertyFunction("$(prop.LastIndexOf('y', 7))", "prop", "x-x-y-y-y-z", "6");
```

Comment:
I totally understood how this overload worked and didn't have to look it up to figure out how that could be a reasonable answer. Completely.

😳